lacework/readme.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Software Composition Analysis with Lacework
```

Comment:
**(nitpick)**: Can we use sentence casing to match others and replace "with" with "using" to match some of the other templates.

```suggestion
title: Software composition analysis using Lacework
```